core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -155,6 +162,46 @@ sealed trait TLVFactory[+T <: TLV] extends Factory[T] {
   }
 }
 
+case class NormalizedString(private val str: String) {
```

Comment:
`extends NetworkElement`?  It seems it's pretty important that we serialize these correctly for when we are passing outcomes over the wire if I understand correctly